potato/activelearning.py

```diff
@@ -0,0 +1,173 @@
+def actively_learn():
+    global user_to_annotation_state
+    global instance_id_to_data
+
+    if "active_learning_config" not in config:
+        logger.warning(
+            "the server is trying to do active learning " + "but this hasn't been configured"
+        )
+        return
+
+    al_config = config["active_learning_config"]
+
+    # Skip if the user doesn't want us to do active learning
+    if "enable_active_learning" in al_config and not al_config["enable_active_learning"]:
```

Comment:
Personally, I find pydantic models and dataclasses easier than checking if statement by if statement if the object is what we expect in the configuration. Then you can do verification in one line and the schema is outlined with types.